xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h

```diff
@@ -114,7 +114,7 @@
 #define ADDON_INSTANCE_VERSION_PERIPHERAL_DEPENDS     "addon-instance/Peripheral.h" \
                                                       "addon-instance/PeripheralUtils.h"
 
-#define ADDON_INSTANCE_VERSION_PVR                    "5.7.0"
+#define ADDON_INSTANCE_VERSION_PVR                    "5.7.1"
```

Comment:
Note: All PVR addons need to be recompiled @MilhouseVH

Reply:
Not sure whether not changing min version min actually leads to no incompatibilities. Maybe I'm wrong on this. I always get puzzled about this min version stuff. @AlwinEsch ?

Reply:
It used to but i'm not sure about current state. Remember it used to be in addon.xml that made addon manager disable it?

Reply:
Just tested. Only increasing version number, but keeping min version number, like @peak3d did, is working just fine. Addon stays compatible, no need to recompile. All good. Sorry for the noise.

Reply:
For a thing like them is not needed to increase min version. There is nothing changed in the "C" Interface, only a new identification string for properties added. All older compiled addons are still compatibel with the change here.